source/javascripts/terminology_tooltip.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+[...document.getElementsByClassName('terminology')].forEach($terminology => {
+    const horizontalMargin = 20;
+
+    const topMargin = document
+        .getElementsByClassName('site-header')[0]
+        .clientHeight;
+
+    const terminology = $terminology.innerText;
+
+    const debug = message => {
+        console.debug(`Terminology tooltip (${terminology}): ${message}`);
+    }
+
+    const $tooltip = $terminology.querySelector('.terminology-tooltip');
```

Comment:
While unlikely, this can be undefined if a class of "terminology" is added to an element that is not using the term tag

Reply:
Yep. It will be a NPE. It will throw in the console. It will not reposition the tooltip. That's good enough for me.

Do you want me to check for `$tooltip` existence and explicitly throw, instead of the implicit NPE?

Reply:
I think we can just return if the queryselector returns nothing for this

Reply:
> I think we can just return if the queryselector returns nothing for this

That would silence a most-likely error on the programmer part -> someone forgetting to grep when attaching the class name in a new context.

I know, in theory, that could be possible -> but I'd strongly advise against that, as it could lead to unexpected results (like "their" new code messing with these tooltips).